node/src/p2p/swarm_manager.rs

```diff
@@ -34,7 +34,8 @@ const BOOTNODE_PROTECT_TAG: u32 = 0;
 const FULL_PROTECT_TAG: u32 = 1;
 const ARCHIVAL_PROTECT_TAG: u32 = 2;
 
-const PEER_HEALTH_CHECK_INTERVAL: Duration = Duration::from_secs(60);
+const PEER_HEALTH_CHECK_INTERVAL: Duration = Duration::from_secs(10 * 60);
```

Comment:
10 mins are quite a lot, but it is needed only if there are no events from peer tracker, no?